presto-product-tests-launcher/bin/run-product-tests

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```

Comment:
Sorry, I don't find it simpler. Yet another layer to me makes it a bit more complex.